build_tools/github/check_build_trigger.sh

```diff
@@ -4,9 +4,10 @@ set -e
 set -x
 
 COMMIT_MSG=$(git log --no-merges -1 --oneline)
+BRANCH_NAME=$(echo "${GITHUB_REF##*/}")
 
 # The commit marker "[cd build]" will trigger the build when required
-if [[ "$GITHUB_EVENT_NAME" == push ||
+if [[ "$GITHUB_EVENT_NAME" == push && "$BRANCH_NAME" != master ||
       "$GITHUB_EVENT_NAME" == schedule ||
       "$COMMIT_MSG" =~ \[cd\ build\] ]]; then
     echo "::set-output name=build::true"
```

Comment:
I think the following is enough:

```suggestion
if [[ "$GITHUB_EVENT_NAME" == schedule ||
      "$COMMIT_MSG" =~ \[cd\ build\] ]]; then
    echo "::set-output name=build::true"
```

whenever there is a `[cd build]` commit marker (in a PR or a push event), we trigger the wheel builder.

Reply:
Yeah, it is easier. We should then remember to add the `[cd build]` commit trigger before pushing to release branches.

Reply:
Yes but it's easier to reason about how to use this.